packages/cli/src/index.ts

```diff
@@ -76,6 +77,19 @@ const reportMetrics = async (req: http.IncomingMessage, res: http.ServerResponse
   res.end(await register.metrics())
 }
 
+const hasBootnodeENRPrefix = (enr) => {
```

Comment:
could we add import the following
```
import { Enr } from './rpc/schema/types.js'
```
and then explicitly mention the parameter type for the following function and do the same for the `addBootNode` function?
```
const hasBootnodeENRPrefix = (enr: Enr) => {
```

Reply:
I would move these functions to a separate `util.ts` file in the same folder level as this `index.ts` and then add your imports there.

Reply:
You also need to add appropriate types for each parameter here and in the `addBootNode` helper.

Reply:
Also, can you rename this to `isValidEnr` or something similar so the name matches what the function is doing?